zerver/openapi/zulip.yaml

```diff
@@ -18300,6 +18300,30 @@ paths:
                           before using this API key.
 
                           **Changes**: Added in Zulip 4.0 (feature level 47).
+                      tenor_api_key:
+                        type: string
+                        description: |
+                          Present if `tenor` is present in `fetch_event_types`.
+
+                          It appears that the usage of API keys on the client side isn't
+                          risky based the examples in the [Tenor Endpoints documentation](https://developers.google.com/tenor/guides/endpoints).
```

Comment:
We should frame this with more certainty. "Tenor API keys are expected to be sent to clients, see the examples in the ..."